ansible/playbooks/aix.yml

```diff
@@ -439,30 +439,48 @@
       # Install X11 extensions                                                #
       # x11.adt.ext installp download requiring an IBMid                      #
       # https://www-01.ibm.com/support/docview.wss?uid=isg1fileset-1198156818 #
+      # x11.vfb installp download requiring an IBMid                          #
+      # http://www-01.ibm.com/support/docview.wss?uid=isg1fileset-2054486724  #
       #########################################################################
+
         - name: Check if X11.adt.ext is installed
           shell: lslpp -l X11.adt.ext >/dev/null 2>&1
           register: does_X11_adt_ext_exist
           ignore_errors: yes
-        - name: Transfer X11 installp to remote host
-          copy:
-            src: /Vendor_Files/aix/adt/X11.adt
-            dest: /tmp/X11.adt
-          when: does_X11_adt_ext_exist.rc != 0
+
+        - name: Check if X11.vfb is installed
+          shell: lslpp -l X11.vfb >/dev/null 2>&1
+          register: does_x11_exist
+          ignore_errors: yes
+          tags: x11
+
+        - name: Transfer and extract X11
+          unarchive:
+            src: /Vendor_Files/aix/OpenGL_X11.tar
+            dest: /tmp
+            remote_src: no
+          when: does_x11_exist.rc != 0 or does_X11_adt_ext_exist.rc != 0
           tags: x11
 
         - name: Install IBM X11 Extensions - installp
-          command: installp -aXYgd /tmp/X11.adt X11.adt.ext
+          command: installp -aXYgd /tmp/x11/X11.adt X11.adt.ext
           register: result.x11
           ignore_errors: yes
           when: does_X11_adt_ext_exist.rc != 0
           tags: x11
 
-        - name: Remove X11 installp
+        - name: Install X11.vfb - installp
+          command: installp -agXYd /tmp/x11 X11.vfb
```

Comment:
Is there a reason not to make this explicit i.e. `/tmp/x11/X11.vfb` to make it consistent with the `installp` command to install `X11.adt.ext`?

Reply:
@sxa555 It will not work that way, the syntax for installing vfb is different